python/src/gluon_ir.cc

```diff
@@ -864,7 +865,12 @@ void init_gluon_ir(py::module &&m) {
       .def("create_lds_barrier_arrive",
            [](GluonOpBuilder &self, Value memDesc, int count) -> Value {
              return self.create<ttag::ArriveBarrierOp>(memDesc, count);
-           });
+           })
+      .def("create_warp_pipeline_border", [](GluonOpBuilder &self) {
+        auto border = self.create<ROCDL::SchedBarrier>(0);
+        border->setAttr("triton.warp_pipeline.border",
```

Comment:
Maybe use a prefix of `gluon.amd.` instead of `triton.` to be more scoped and specific for now.